Dependency not found with node 4.2.1 and npm 2.14.7

I'm using stylelint-selector-bem-pattern with a .stylelintrc and the stylelint CLI. When running my stylelint tests I get the following error:

```
> ismaywolff.nl@0.1.0 test:stylelint /Users/ismay/Projects/ismay.github.io
> stylelint '{src,lib}/**/*.css'

Error: Cannot find module 'postcss/lib/result'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/ismay/Projects/ismay.github.io/node_modules/stylelint-selector-bem-pattern/index.js:2:14)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
```

Code that resulted in this error can be found in this [branch](https://github.com/ismay/ismay.github.io/tree/lint-bem). 
  * [.stylelintrc](https://github.com/ismay/ismay.github.io/blob/lint-bem/.stylelintrc)
  * [package.json](https://github.com/ismay/ismay.github.io/blob/lint-bem/package.json)

When I install postcss manually with `npm i postcss` the error disappears, but that shouldn't be necessary right? Am I doing something wrong, or is this a bug?

--- a/package.json
+++ b/package.json
@@ -30,12 +30,12 @@
   "homepage": "https://github.com/davidtheclark/stylelint-selector-bem-pattern#readme",
   "dependencies": {
     "lodash": ">=3.10.0",
+    "postcss": "^5.0.19",
     "postcss-bem-linter": "^2.1.0",
     "stylelint": ">=3.0.2"
   },
   "devDependencies": {
     "eslint": "^1.10.3",
-    "postcss": "^5.0.10",
     "stylelint-rule-tester": "^0.6.1"
   }
 }

